akka-actor/src/main/resources/reference.conf

```diff
@@ -411,7 +411,11 @@ akka {
         # work distribution approach is disabled and only the hash based is
         # used irrespective of the number of unique actors. Valid range is
         # 0 to 2048 (inclusive)
-        fair-work-distribution-threshold = 128
+        queue-selector = "akka.dispatch.affinity.FairDistributionHashCache"
+
+        # When using the "akka.dispatch.affinity.FairDistributionHashCache" queue selector
+        # the threshold is how many Runnables' indicies it will cache at most.
+        fair-work-distribution.threshold = 128
```

Comment:
The rename was intentional, I've updated the benches and corrected the indices typo. :+1: